apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -1,42 +1,333 @@
+                      lineHeight: '28px',
+                      minHeight: 'auto',
+                      height: 'auto',
+                      width: '100%',
+                      textOverflow: 'ellipsis',
+                      '&:not(:placeholder-shown)': {
+                        borderStyle: 'none',
+                        padding: 10,
+                      },
+                      '&:hover, &:focus': {
+                        borderStyle: 'solid',
+                        padding: 9,
+                      },
+                      '&:disabled': {
+                        backgroundColor: colorScheme === 'dark' ? colors.B15 : theme.white,
+                        color: colorScheme === 'dark' ? theme.white : theme.black,
+                        opacity: 1,
+                      },
+                    },
+                  })}
+                  {...field}
+                  value={field.value !== undefined ? field.value : selectedProvider.displayName}
+                  // error={showErrors && fieldState.error?.message}
+                  type="text"
+                  data-test-id="title"
+                  placeholder="Enter workflow name"
+                  // disabled={readonly}
+                />
+              );
+            }}
+          />
+          <ActionIcon
+            variant={'transparent'}
+            onClick={() => {
+              navigate('/integrations');
+            }}
+          >
+            <Close color={colors.B40} />
+          </ActionIcon>
+        </Group>
+        <Group mb={16} spacing={16}>
+          <IntegrationChannel name={CHANNEL_TYPE_TO_STRING[selectedProvider.channel]} type={selectedProvider.channel} />
+          <IntegrationEnvironmentPill
+            name={
+              environments?.find((environment) => environment._id === selectedProvider.environmentId)?.name ||
+              'Development'
+            }
+          />
+        </Group>
+        <CenterDiv>
+          <ActiveWrapper active={isActive}>
+            <Controller
+              control={control}
+              name="active"
+              render={({ field }) => <Switch checked={isActive} data-test-id="is_active_id" {...field} />}
+            />
+            <StyledText data-test-id="connect-integration-form-active-text">
+              {isActive ? 'Active' : 'Disabled'}
+            </StyledText>
+          </ActiveWrapper>
+          <Controller
+            control={control}
+            name="identifier"
+            render={({ field }) => (
+              <InputWrapper>
+                <Input
+                  label="Instance key"
+                  error={errors.identifier?.message}
+                  rightSection={
+                    <CopyWrapper onClick={() => identifierClipboard.copy(field.value)}>
+                      {identifierClipboard.copied ? <Check /> : <Copy />}
+                    </CopyWrapper>
+                  }
+                  {...field}
+                />
```

Comment:
We need to add validation and errors on this one. [figma](https://www.figma.com/file/zkLXTTP8Ds57ZciHe1hrkx/Integration-store?type=design&node-id=1180-84049&t=ywcdm5NpPQCjiv3g-0)